### PR TITLE
#5611 Internationalize CRSSelector

### PR DESCRIPTION
**Pull request by jomue**
## Description

This pull requests adds internationalization support to the CRSSelector, which currently contains hardcoded English strings.

I've added the existing English strings to all language files, so this shouldn't change anything for the users. I've also added German translations.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix
 - [x] Other...: Internationalization

## Issue

**What is the current behavior?**
#5611
The CRSSelector is always shown in English, even after changing to another language.

**What is the new behavior?**

The CRSSelector can be translated. The following screenshot shows the German translation, which is included in this PR.

image

## Breaking change
**Does this PR introduce a breaking change?**
 - [x] No

## Other useful information

I'm new to MapStore and relatively new to React, so I hope I've done things properly :smiley:

### CODE REVIEW

File: web/client/translations/data.it-IT.json [hunk 1]
```diff
@@ -160,6 +160,9 @@
         "elevationNotAvailable": "Non disp.",
         "mapScale": "Scala:",
         "showMousePositionCoordinates": "Mostra/Nascondi Coordinate",
+        "showCrsSelector": "Select projection",
+        "crsSelectorFilterPlaceholder": "Filter projection",
+        "crsSelectorSelectedCRS": "Selected:",
```
Comment (MV88) [inline web/client/translations/data.it-IT.json:165]: please provide italian translations

File: web/client/translations/data.fr-FR.json [hunk 2]
```diff
@@ -160,6 +160,9 @@
         "elevationNotAvailable": "N/A",
         "mapScale": "Échelle :",
         "showMousePositionCoordinates": "Montrer/cacher les coordonnées",
+        "showCrsSelector": "Select projection",
+        "crsSelectorFilterPlaceholder": "Filter projection",
+        "crsSelectorSelectedCRS": "Selected:",
```
Comment (MV88) [inline web/client/translations/data.fr-FR.json:165]: please provide french translations

File: web/client/translations/data.es-ES.json [hunk 3]
```diff
@@ -160,6 +160,9 @@
         "elevationNotAvailable": "N/P",
         "mapScale": "Escala:",
         "showMousePositionCoordinates": "Mostrar/ocultar las coordenadas",
+        "showCrsSelector": "Select projection",
+        "crsSelectorFilterPlaceholder": "Filter projection",
+        "crsSelectorSelectedCRS": "Selected:",
```
Comment (MV88) [inline web/client/translations/data.es-ES.json:165]: please provide spanish translations

**Comment by jomue** [comment]
@MV88: I'd really like to provide more translations, but I don't speak any of those languages, unfortunately.

**Comment by MV88** [comment]
> @MV88: I'd really like to provide more translations, but I don't speak any of those languages, unfortunately.

no problem, myself too sometimes use google translator for this purpose :)
going to merge and fix later these translations.

thanks for your time

**Comment by MV88** [comment]
closed in favor of https://github.com/geosolutions-it/MapStore2/pull/5614

**Pull request closed without merging**